### ISSUE TITLE
accessibility - challenge-buttons are not accessible via keyboard

### ISSUE THREAD

**Issue by sarbbottam** [FreeCodeCamp/FreeCodeCamp#8575]
#### Challenge Name
https://www.freecodecamp.com/challenges/learn-how-free-code-camp-works

#### Issue Description
Try pressing the `tab` key to reach the `button`, they never receives focus.
Currently the challenge buttons are implemented as `div`, they should be `button`.

#### Browser Information
* Browser Name, Version: Any
* Operating System: Any
* Desktop & Mobile & Tablet

#### Screenshot
![screen_shot_2016-05-11_at_10_22_05_am](https://cloud.githubusercontent.com/assets/949380/15190077/e1bd6f58-1762-11e6-9072-029c6d4f227f.png)
